src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -5749,3 +5749,77 @@ def test_aks_update_with_defender(self, resource_group, resource_group_location)
         # delete
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
+
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_update_with_azuremonitormetrics(self, resource_group, resource_group_location):
```

Comment:
Triggered on [pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=61745796&view=results) to run the test.

Reply:
Copied over the results from the [successful run](https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=61805764&view=results) in the recording folder.